src/main/java/core/basesyntax/strategy/Operation.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.Transaction;
+
+public interface Operation {
+    void execute(Transaction transaction);
```

Comment:
we can't have enum and this class named identically, this is a handler for a strategy, so:
```suggestion
public interface OperationHandler {
    void handle(Transaction transaction);
```